apps/resources/scala3-doc.json

```diff
@@ -4,7 +4,7 @@
     "central"
   ],
   "dependencies": [
-    "org.scala-lang:scala3-doc_3.0.0-M3:3.0.0-M3"
+    "org.scala-lang:scala3-doc_3.0.0-RC1:3.0.0-RC1"
```

Comment:
@romanowski is this correct? What should the main class name be instead of `dotty.tools.dottydoc.Main`?

Reply:
in RC1 we've moved to calling it just `scaladoc`. It should use `dotty.tools.scaladoc.Main` for cmd line version for sbt-like it is `dotty.tools.dottydoc.Main`